pallets/issue/src/lib.rs

```diff
@@ -262,12 +309,54 @@ pub mod pallet {
 			Self::deposit_event(Event::IssuePeriodChange { period });
 			Ok(().into())
 		}
+
+		#[pallet::weight(<T as Config>::WeightInfo::set_issue_period())]
+		#[transactional]
+		pub fn rate_limit_update(
+			origin: OriginFor<T>,
+			limit_volume_amount: Option<BalanceOf<T>>,
+			limit_volume_currency_id: T::CurrencyId,
+			current_volume_amount: BalanceOf<T>,
+			interval_length: T::BlockNumber,
+			last_interval_index: T::BlockNumber,
```

Comment:
I think I would remove `current_volume_amount` and `last_interval_index` from this extrinsic actually. I don't think that being able to assign these two in this extrinsic gives us a benefit, in fact, I would even argue that having to overwrite the `current_volume_amount` with some value here is bad, because:
Let's assume we don't want to change it and leave the value that is used currently on-chain intact. Now, with the current implementation, we have to provide something for the `current_volume_amount`. We can choose the current on-chain value for that. But by the time we call and finalize this extrinsic, that value might have changed and then we reset it to some earlier value.